src/app/person/gender.hpp

```diff
@@ -5,7 +5,9 @@
 enum class Gender
 {
     Male,
-    Female
+    Female,
+    Unknown
```

Comment:
correct to explicit values, 0/1/255? if derived from std::uint8_t

Reply:
done